onboarding/src/main/java/com/toggl/onboarding/signup/ui/SignUpFragment.kt

```diff
@@ -36,6 +44,24 @@ class SignUpFragment : Fragment(R.layout.fragment_signup) {
         login_button.setOnClickListener {
             store.dispatch(SignUpAction.GoToLoginTapped)
         }
+
+        val defaultCriteriaColor = ContextCompat.getColor(requireContext(), R.color.authentication_text)
+        val errorCriteriaColor = ContextCompat.getColor(requireContext(), R.color.error)
+
+        val isPasswordStrongChannel = Channel<Boolean>()
+
+        passwordStrengthWatcher = password.doAfterTextChanged { isPasswordStrongChannel.offer(it.representsStrongPassword()) }
+
+        isPasswordStrongChannel
+            .consumeAsFlow()
```

Comment:
Why is this at all needed when the password already is in the state? 🤔 

```suggestion
        store.state
            .map { it.password is Password.Strong }
```